### PR TITLE
[Docs] Adding AUTHORS and CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This is the list of Material Components for Flutter authors for copyright
+# purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+Google Llc
```
Comment (DaveShuckerow) [inline AUTHORS]: Should this be 'Google LLC'?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Please see the [Material Components code of conduct](https://github.com/material-components/material-components/blob/develop/CODE_OF_CONDUCT.md).
```
Comment (DaveShuckerow) [inline CODE_OF_CONDUCT.md]: Should the language here be more along the lines of

'The MC COC applies to this repository'?

If this is our standard language for sharing licenses, codes of conduct, etc, then this is fine with me.

Comment (willlarche) [reply CODE_OF_CONDUCT.md]: This is the standard language. Matches [https://github.com/material-components/material-components-ios/blob/develop/CODE_OF_CONDUCT.md](https://github.com/material-components/material-components-ios/blob/develop/CODE_OF_CONDUCT.md)

File: AUTHORS (hunk 1 above)
Comment (willlarche) [reply AUTHORS]: Done.

**Comment by willlarche** [comment]
PTAL

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (DaveShuckerow) [reply CODE_OF_CONDUCT.md]: Sounds good to me

**Pull request merged**